examples/morph.html

```diff
@@ -29,11 +29,11 @@
             fullscreen: true
           }).appendTo(document.body);
 
-          Two.Resolution = 32;
-
-          var blob = two.makeCircle(two.width / 2, two.height / 2, two.height / 3);
+          var blob = two.makeCircle(two.width / 2, two.height / 2, two.height / 3, 32);
```

Comment:
New Two.js' circles have 4 vertices by default and are not automatically smoothed, like old Two.js' circles were.